src/main/holochainManager.ts

```diff
@@ -157,36 +156,30 @@ export class HolochainManager {
     const conductorEnvironmentPath = launcherFileSystem.conductorEnvironmentDir(partitionName);
     const configPath = launcherFileSystem.conductorConfigPath(partitionName);
     console.log('configPath: ', configPath);
+    const generateConductorConfig = fs.existsSync(configPath)
+      ? rustUtils.overwriteConfig
+      : rustUtils.defaultConductorConfig;
+    const conductorConfig = generateConductorConfig(
+      adminPort,
+      lairUrl,
+      bootstrapUrl || DEFAULT_BOOTSTRAP_SERVER,
```

Comment:
I think this logic is not how we want it. If the config file already exists, the bootstrap and signaling urls should not be overwritten with the default bootstrap server, i.e. they should be passed as undefined here (which would be `None` on the Rust side). Otherwise if someone deliberately manually edits the bootstrap and signaling servers in the config file it will always be overwritten by the launcher.